ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -657,7 +657,8 @@ private void DetachViewFromInstance(ReactRootView rootView, IReactInstance react
                 var coreModulesPackage = new CoreModulesPackage(
                     this,
                     InvokeDefaultOnBackPressed,
-                    _uiImplementationProvider);
+                    _uiImplementationProvider,
+                    true);
```

Comment:
>true [](start = 20, length = 4)

Parameterize on the ReactInstanceManagerBuilder.

Reply:
Is this like a note to yourself?

---
In reply to: [174644781](https://github.com/Microsoft/react-native-windows/pull/1717#discussion_r174644781) [](ancestors = 174644781)

Reply:
Yeah - I really should start PRing against my own fork first :-)

---
In reply to: [174843581](https://github.com/Microsoft/react-native-windows/pull/1717#discussion_r174843581) [](ancestors = 174843581,174644781)